addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -29,6 +29,10 @@ $-forum-sidebar-width: map-get($container-max-widths, sm) / 2;
         .o_searchbar_form {
             min-width: 200px;
         }
+        .dropdown-menu .s_share {
+            width: max-content;
+            padding: 1rem;
```

Comment:
as discussed in the review msg, removed this and replace by "None" layout + here make it centered?